mysql-test/suite/sys_vars/r/all_vars.result

```diff
@@ -173,20 +173,105 @@ terminology_use_previous
 tls_ciphersuites
 tls_ciphersuites
 use_secondary_engine
+wsrep_OSU_method
```

Comment:
Why this test needed re-record of the result? I mean: how did it work before? These variables should have been listed earlier as well, so it should have failed.
Or do I miss something?